### PR TITLE
Add release script

### PR DESCRIPTION
**Pull request by marvinhagemeister**
```bash
$ ./release
Enter new version number (current: 10.0.84):
10.0.85
```

After pressing `Enter`, the version will be changed in `package.json` and ultimately published to npm.

### CODE REVIEW

File: release [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```
Comment (phihag) [inline release:1]: There's no bash-specific code here, so we can just use `#!/bin/sh`.

File: release [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+set -e
+
+CURRENT_VERSION=$(node -e 'console.log(require("./package.json").version)')
+echo "Enter new version number (current: $CURRENT_VERSION):"
+read VERSION;
+git commit -am "release v$VERSION" && git tag "v$VERSION" && git push && git push --tags && npm publish
```
Comment (phihag) [inline release:7]: Maybe `set -x` here to see the commands that are run

Comment (phihag) [inline release:7]: Instead of `&&`, every command can be split onto its own line.

File: release [hunk 3]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+set -e
+
+CURRENT_VERSION=$(node -e 'console.log(require("./package.json").version)')
+echo "Enter new version number (current: $CURRENT_VERSION):"
+read VERSION;
```
Comment (phihag) [inline release:6]: The `;` doesn't do anythingg here, can be removed

File: release (hunk 1 above)
Comment (marvinhagemeister) [reply release]: Good idea! Shell scripting is not my strong suite for sure 😬

**Pull request merged**